package.json

```diff
@@ -26,22 +26,22 @@
     "node-gyp-build": "^4.8.0"
   },
   "peerDependencies": {
-    "tree-sitter": "^0.21.1"
+    "tree-sitter": "^0.21.0"
   },
   "peerDependenciesMeta": {
     "tree_sitter": {
       "optional": true
     }
   },
   "devDependencies": {
-    "tree-sitter-cli": "^0.22.2",
+    "tree-sitter-cli": "^0.23.0",
     "prebuildify": "^6.0.0"
   },
   "scripts": {
-    "build": "tree-sitter generate --no-bindings",
-    "prestart": "tree-sitter build --wasm -o tree-sitter-r.wasm",
-    "start": "tree-sitter playground",
-    "test": "tree-sitter test",
+    "build": "npx tree-sitter-cli generate --no-bindings",
+    "prestart": "npx tree-sitter-cli build --wasm -o tree-sitter-r.wasm",
+    "start": "npx tree-sitter-cli playground",
+    "test": "npx tree-sitter-cli test",
```

Comment:
The `tree-sitter` invocations were changed to make the scripts a bit more robust. Calling `tree-sitter` through `npx` like this ensures the correct version from the dev dependencies is used, and makes the scripts independent from the local system state.

Reply:
This should probably be changed upstream then?

Reply:
Apologies. This was a misunderstanding on my part. It turns out NPM puts the dev dependency binaries on the path when running scripts, so this was correct. But I see you didn't keep this change anyway :+1: